packages/runtime/src/enhancements/utils.ts

```diff
@@ -62,7 +62,9 @@ function loadPrismaModule(prisma: any) {
     if (prisma._engineConfig?.datamodelPath) {
         const loadPath = path.dirname(prisma._engineConfig.datamodelPath);
```

Comment:
Hi @Azzerty23 , thanks a lot for making this PR!

I'm trying to reproduce the issue with the repo you shared (express-api project) but things seem to work well from my environment. I added a line of log to print the `prisma._engineConfig?.datamodelPath` and it's like:

    `<root>/node_modules/.prisma/client/schema.prisma`

, and loading the module from that folder works (since it contains Prisma generated code there)...

So I'm still struggling to understand what could have caused failure on your side. Do you mind adding a similar line of log in your environment and seeing what it prints? Thanks!

Reply:
Thank you so much! I think I have understood the issue. In my build script, I copy the `schema.prisma` to my `dist/` folder to avoid this warning:
<img width="806" alt="Capture d’écran 2023-06-05 à 14 45 21" src="https://github.com/zenstackhq/zenstack/assets/43639468/f8bf96a4-8502-45a3-a0d7-282a0fd39783">
Solution from: https://github.com/prisma/prisma/issues/10512#issuecomment-1254779466

Copying the `schema.prisma` to my `dist/` folder prevents `prisma._engineConfig?.datamodelPath` from recognizing the correct path for the Prisma client.

Indeed, when I console.log `prisma._engineConfig?.datamodelPath`, I get:
```
/Users/augustin/Documents/Dev/Test/zenstack-rest-turbo/apps/express-api/dist/schema.prisma
```
That's why I only had this issue with my built project. However, I don't understand why you didn't encounter the same problem with my repo. Ideally, I would like to find a solution to avoid the warning altogether but it's not blocking.